Source/BugsnagConfiguration.m

```diff
@@ -249,6 +251,22 @@ - (NSDictionary *)sessionApiHeaders {
              };
 }
 
+- (void)setEndpointsForNotify:(NSString *_Nonnull)notify sessions:(NSString *_Nonnull)sessions {
```

Comment:
should we support providing a null sessions value? It seems on the other notifiers that is something that we support

Reply:
No - these annotations are part of the original design spec.